### PR TITLE
Feature: Add Skip-termination Command to keep debug instance up

### PR DESCRIPTION
**Pull request by luis-falcon**
Feature:

This PR adds a Parameter (--skip-termination) to be able to leave the debug instance running for further debugging. 
This PR also adds the (--terminate-debug) that will terminate the debug instance that was left running.

### CODE REVIEW

**Review by mjlshen — APPROVED** [review]
/lgtm

File: cmd/egress/cmd.go
Comment (mjlshen) [inline cmd/egress/cmd.go:46]:
```suggestion
	skipAWSInstanceTermination bool
	terminateDebugInstance     string
```
We may want to make these private configuration options just like the rest

**Review by boranx — APPROVED** [review]
/approve
/hold

lgtm 👍, feel free to `unhold` when you're ready to merge

**Pull request merged**
